packages/expression-evaluator/lib/evaluators/OrderByEvaluator.ts

```diff
@@ -0,0 +1,109 @@
+import type * as RDF from '@rdfjs/types';
+import type * as E from '../expressions';
+import type { RegularFunction } from '../functions';
+import type { IAsyncEvaluatorContext } from './ContextualizedEvaluator';
+import { ContextualizedEvaluator } from './ContextualizedEvaluator';
+
+export class OrderByEvaluator extends ContextualizedEvaluator {
```

Comment:
Perhaps we can already design towards bus-ifying it later on?
I remember us discussing there being a comparator factory bus, with this being a term-comparator.

In that sense, _extending_ `ContextualizedEvaluator` might not be a good approach.
Instead, composition might be better, i.e., just saving it as internal field.

Reply:
ah yes! I forgot about that one. Sure!